05_streaming/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>04_Frontend</title>
```

Comment:
Very minor, but "04_Frontend" appears in a few places inside the 05 and 06 directories.  It very well may have been this way before too :D

Reply:
No, that was me. Fixed!